docker-compose.yml

```diff
@@ -49,46 +68,67 @@ services:
     volumes:
       - ./dev/database/auth:/var/lib/mysql
 
-  eureka-service:
+  profile_microservice:
+    container_name: profile_microservice
+    depends_on:
+      - profile_mysql
+      - eureka-service
+      - gateway
+    # Path to teh root dir of teh microservice (where teh Dockerfile is)
     build:
-      context: ./Microservices/service-registry
-      dockerfile: Dockerfile
+      context: ./Microservices
+      dockerfile: Profile/Dockerfile
+      args:
+
+        # name of teh folder in Microservices
+        SERVICE_PATH: "Profile"
+    # pass in the env file to be used
     env_file: ".env"
     environment:
+      PROFILE_DB_IP: profile_mysql
       SERVICE_REGISTRY_IP: eureka-service
+    ports:
+      # ideally it would be mapped to a more static internal port
+      # - "${PROFILE_MICROSERVICE_PORT}:8080"
```

Comment:
why not do this? if theres a solid reason for not then id say remove these comments